docs/FAQ.md

```diff
@@ -95,7 +95,7 @@ which may require multiple invocations of `jj rebase -r` or `jj rebase -s`.
 
 To squash or split commits, use `jj squash` and `jj split`.
 
-### How can I keep my scratch files in the repository?
+### How can I create scratch files without committing them to the repository?
```

Comment:
Nit: "keep or create [...]", since initially it was all about keeping Git's untracked files available.